### ISSUE TITLE
Improve answer scoring on fine-tuned Bert withSQUAD

### ISSUE THREAD

**Issue by zharenkov** [google-research/bert#783]
Hi,
I've got a question - is there any options to change/fix scoring for Squad fine-tuned model.
E.g. we are feeding to predictor 5 examples. Same question and 5 different paragraphs. The first paragraph contains correct answer for this question, other 4 - not.
Q1 - P1, Q1 - P2, Q1 - P3, Q1 - P4, Q1 - P5
But when Bert calculates logits, the start_logit+end_logit for not relevant paragraphs is higher than for the right one. 
Maybe I missed some settings or can try to change some training code to achieve more precise results?
Thanks!

**Comment by luke4u** [comment]
Hi zharenkov, hope you are well. 

I am thinking the same question you have.  The problem is Bert's limit on token length. If breaking down the paragraph into chunks, you have answers from different chunks with different scores. Not sure if sensible to compare these scores. Any chance to share some insights?

Thanks. Luke
